### ISSUE TITLE
Raspiblitz not booting (press Control-D to continue)

### ISSUE THREAD

**Issue by radar24** [raspiblitz/raspiblitz#1587]
my raspiblitz has been running 24/7 and i just found out that it has been frozen few days.. 
I rebooted the system and it gives me this and raspiblitz seem not working... 
does this mean i'm getting dead sd card? and i have to re setup all over again?

![IMG_5304](https://user-images.githubusercontent.com/34410574/94119914-77f97300-fe8a-11ea-8d02-2bac5de1f017.JPG)

**Comment by openoms** [comment]
Flashing a new SDcard does not mean to set up all over again, it will restore your wallets , channels and settings form the disk.

You might as well try to reboot by pulling the power and/or attach a keyboard and press CTRL+D.

The cause of this is yet unknown to me, but flashing a new SD solves it for good. Use a new card if possible.

**Comment by ChuckNorrison** [comment]
Hey, if you have any issues on this, check out my way on this root password state:
https://github.com/rootzoll/raspiblitz/issues/1551

Still dont know how this happens exactly

tl;dr
lnd-rescue file works great, check out the backup docs on the raspiblitz

**Comment by amir0ff** [comment]
same issue #1618

**Comment by openoms** [comment]
This happens to me often after a cold restart (pulling the power) but can't say that always (also reluctant to test just for the sake of it).
Normal restarts are usually ok, but happened with that as well.
What helps:
- attach a USB keyboard and press CTRL +D as the screen says
- reflashing the SDcard solved it (takes a long time with recovering many services)

**Comment by ChuckNorrison** [comment]
Happens to me 4 times. Last time right now after Backup LND rescue file via menu and shutdown.

I observed a corrupted css file on my RTL webinterface and decided to backup the lnd. This was running on v.1.6.0 default without any modifications except OpenVPN Client setup and some ufw rules to get it work.

After ctrl+D it is possible to ssh. Here some logs: https://termbin.com/u927

I updated from patch menu to latest v1.6.0 and ends up in Control-D again

i checked the bitcoin.conf which was missing in the logs... seems fine to me

```
admin@raspberrypi:/home/bitcoin/.bitcoin $ ls -laht
total 26M
drwxr-xr-x  2 bitcoin bitcoin  76K Oct 12 17:28 chainstate
-rw-------  1 bitcoin bitcoin 8.2K Oct 12 17:28 debug.log
drwx------  3 bitcoin bitcoin 4.0K Oct 12 17:26 wallet.dat
drwxr-xr-x  8 bitcoin bitcoin 4.0K Oct 12 17:26 .
-rw-------  1 bitcoin bitcoin    5 Oct 12 17:26 bitcoind.pid
-rw-------  1 bitcoin bitcoin 243K Oct 12 17:11 fee_estimates.dat
-rw-------  1 bitcoin bitcoin 1.2K Oct 12 17:11 banlist.dat
-rw-------  1 bitcoin bitcoin 3.9M Oct 12 17:11 peers.dat
drwxr-xr-x 12 root    root    4.0K Oct 12 17:05 ..
-rw-------  1 bitcoin bitcoin  22M Oct 12 15:33 mempool.dat
drwxr-xr-x  3 bitcoin bitcoin 132K Oct 12 08:44 blocks
-rwxr-xr-x  1 bitcoin bitcoin  546 Oct  8 22:11 bitcoin.conf
drwx------  3 bitcoin bitcoin 4.0K Sep  2 00:10 specter
drwx------  3 bitcoin bitcoin 4.0K Aug 30 16:39 specter_hotstorage
drwx------  3 bitcoin bitcoin 4.0K Aug 15 17:16 indexes
-rw-------  1 bitcoin bitcoin    0 Aug 11 19:54 .lock
```
even the symlink is fine. 

It was no problem to start bitcoind manually after i did all the checks above. A pi reboot results in Control-D again.

The message "bitcoind service is starting" is wrong. A manual start works here. I observed a network connectivity issue while pi was booting (ping ok -> no ping -> ping ok) hangs around 2 mins without connectivity, maybe this is the reason the scripts are not working... race condition while booting?!

After bitcoind was started manually, everything seems to work again except my RTL interface which is still broken and results in bad gateway. I will try to reinstall those services

// i was able to get rid of the Control-D and repair every service manually with reinstallation. The ssh service rejected to start while booting. After boot it was possible to start this service manually. Behaviour like the bitcoind service described above.

Seems the trick for Control-D was to add options `_netdev, nofail` in /etc/fstab for USB HDD

// more details in here: https://github.com/rootzoll/raspiblitz/issues/1618#issuecomment-708661606

**Comment by martinschultzkristensen** [comment]
> Flashing a new **SDcard does not mean to set up all over again**, it will restore your wallets , channels and settings form the disk.
> 
> You might as well try to reboot by pulling the power and/or attach a keyboard and press CTRL+D.
> 
> The cause of this is yet unknown to me, but flashing a new SD solves it for good. Use a new card if possible.

If I download a fresh & flash an image to a new SD card, will my wallet then also restore, or does it have to be a backup img-file?
Thanks for the answer.

![IMG_20201108_175501](https://user-images.githubusercontent.com/27158803/98471400-39c2d380-21ec-11eb-93d9-5e1b39af11a0.jpg)

![IMG_20201108_175650](https://user-images.githubusercontent.com/27158803/98471398-30396b80-21ec-11eb-894b-8308e6d15b42.jpg)

**Comment by openoms** [comment]
>If I download a fresh & flash an image to a new SD card, will my wallet then also restore, or does it have to be a backup img-file

as wrote above "it will restore your wallets, channels and settings from the disk.".

**Comment by martinschultzkristensen** [comment]
> > If I download a fresh & flash an image to a new SD card, will my wallet then also restore, or does it have to be a backup img-file
> 
> as wrote above "it will restore your wallets, channels and settings from the disk.".

Thanks. I just wanted to make sure that it also goes for a new image.

**Comment by aguyisok** [comment]
I've had this issue as well when setting up Raspiblitz, it started when raspberry pi did final reboot after LND setup. I've tried different SD Card, went and bought USB keyboard but for some odd reason keyboard wouldn't get detected. What seems to solve it is plugging HDD into regular USB port, non 3.0. Not sure if it's related or not, but after that problem went away and Raspiblitz started node syncing.. fingers crossed this solved it.

**Comment by ChuckNorrison** [comment]
long time test showed up several tweaks did the trick:
1. /etc/fstab with additional options for /mnt/hdd
`UUID=02fa8fa9-85ff-43b3-a9c2-ae7ee29c064c /mnt/hdd ext4 noexec,defaults,_netdev,nofail 0 2`

2. USB2.0 for external HDD

3. Disable BT and WiFI via /boot/config.txt with: 
`dtoverlay=disable-wifi`
`dtoverlay=disable-bt`

All that stuff works around the power consumption root issue on the Pi 4

// RaspiBlitz v1.6.1 still seems to be affected to that. Did the tweaks after reflashed sd card and recovery installation.

**Comment by aguyisok** [comment]
> long time test showed up several tweaks did the trick:
> 
> 1. /etc/fstab with additional options for /mnt/hdd
>    `UUID=02fa8fa9-85ff-43b3-a9c2-ae7ee29c064c /mnt/hdd ext4 noexec,defaults,_netdev,nofail 0 2`
> 2. USB2.0 for external HDD
> 3. Disable BT and WiFI via /boot/config.txt with:
>    `dtoverlay=disable-wifi`
>    `dtoverlay=disable-bt`
> 
> All that stuff works around the power consumption root issue on the Pi 4
> 
> // RaspiBlitz v1.6.1 still seems to be affected to that. Did the tweaks after reflashed sd card and recovery installation.

I'm a newb.. can you provide step by step from SSH terminal? thx

**Comment by ChuckNorrison** [comment]
as newb, you should wait for the next blitz release or get some google skills up. Just search for the filename together with "raspi pi4". Mostly `sudo nano` is your best friend :)

**Comment by aguyisok** [comment]
gotta start learning somewhere.. ok, i'll see if i can get this figured out.

**Comment by Moonkmeander** [comment]
I have the same issue, minus the ctrl+d option. It just boots up and ends at i2c /dev entries where it ends and stay. I followed the instructions to flash the card using balena and used the image provided in the link -> put the SD card in raspberry and booted. 

However, the raspberry has been used with the default software before this. 

Any clues to what might be wrong here?

**Comment by radar24** [comment]
I'm the OP and after i opened this issue, i couldn't figure this out and i was quite busy with other things so i kinda abandoned my raspiblitz. 3 months later now i finally got time to fix it so i powered on my raspiblitz and it just works now.. i wonder why but it just worked.. syncing three months data took quite a time. my raspiblitz may be needed some resting? i don't know..

**Comment by nyxnor** [comment]
@radar24 Please close this issue.
Also, no issues about this after v1.7 came, seems to be fixed.
